packages/components/api/index.js

```diff
@@ -0,0 +1,17 @@
+import { initializeApiStore } from './cached-apis'
```

Comment:
this should be in the Search package until there is a need to elevate as package

Reply:
Moved into search package